### DISCUSSION TITLE
NEEY 10A Smart Active Balancer: confirmed as working (except for setting balacing current)

### DISCUSSION THREAD

**Discussion by neeu2** [syssi/esphome-jk-bms#570] [General]
All working well except setting the balancing curent. After setting 10A using app, I can modify it to anyting below 4A using esp and HA but anyting above is rejected.
I guess it's designed for 4A model. Just wondering if this possible to modify to accomadate 10A or 15A models.

Many thanks,

**Comment by syssi** [comment]
I wasn't aware of these models. There are 4A, 10A and 15A models available?

**Reply by syssi** [reply]
https://github.com/syssi/esphome-jk-bms/pull/571/files

**Reply by syssi** [reply]
I've pushed a fix to main. Please recompile your YAML. The upper limit is 15A now.

**Reply by neeu2** [reply]
Thank you so much, tested and working perfectly.
Also as a side note, I was able to combine both NEEY BLE and JK-BMS BLE connections using one esp32d1
Thanks again

**Reply by syssi** [reply]
Keep in mind to decrease the log level to `INFO` as soon everything is working. It will free some resources and gives some more headroom for stability.

**Reply by neeu2** [reply]
Strange behaviour, if I cange the logging level to info, certain settings such as Balancing state fail to update, changing back to debug fixed that so kept at Debug level.

**Reply by syssi** [reply]
Interesting. Do you use `mqtt` or the native `api`?

**Reply by syssi** [reply]
This is the root cause: https://github.com/esphome/esphome/pull/7321

**Comment by neeu2** [comment]
yes, very odd. I am using nativve API.
To check, I've changed the logging level to info on other esp connected to JK Bms via rs485 and that basically breaks lot of sensors, they don't get updates and some such as cell voltages took over 5 min to appear. Changing back to debug reverts everything back to fully working and all sensors reporting under 5 sec.
